ambari-server/src/main/resources/stacks/HDP/2.3/upgrades/config-upgrade.xml

```diff
@@ -398,6 +398,7 @@
           <definition xsi:type="configure" id="hdfs_user_agent" summary="Set User-Agent">
             <type>core-site</type>
             <set key="fs.azure.user.agent.prefix" value="User-Agent: APN/1.0 Hortonworks/1.0 HDP/{{version}}" if-type="core-site" if-key="fs.azure.user.agent.prefix" if-key-state="absent" />
+            <set key="fs.gs.application.name.suffix" value=" (GPN:Hortonworks; version 1.0) HDP/{{version}}" if-type="core-site" if-key="fs.gs.application.name.suffix" if-key-state="absent" />
```

Comment:
Do we even support these ancient versions of HDP on Ambari 2.7? I thought that we only supported HDP 2.6+ ...

Reply:
I wanted the new property to be consistent with the related S3 and Azure properties.

I don't know if these versions are officially supported.  I tested with 2.5 -> 2.6 upgrade, and both install and upgrade worked fine.